src/services/requestHandler.js

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+
+/**
+ * This function creates an instance of axios and return
+ * it for making API request
+ * @param { Object} headers an object of http headers
+ * @param { Object } body the request body
+ * @returns { instance } an instance of axios
+ */
+const axiosInstance = (headers, body) => {
```

Comment:
add the default headers to the instance, we don't want to be passing in headers every time we need to make a request